spec/support/standard_setup.rb

```diff
@@ -0,0 +1,161 @@
+class StandardSetup
+  include FactoryGirl::Syntax::Methods
+
+  attr_reader :cases, :user_teams
+  # rubocop:disable Metrics/MethodLength
+
+  def initialize
+    @teams = {
+       disclosure_bmt_team:       find_or_create(:team_dacu),
+       disclosure_team:           find_or_create(:team_dacu_disclosure),
+       responding_team:           create(:responding_team),
+       press_office_team:         find_or_create(:team_press_office),
+       private_office_team:       find_or_create(:team_private_office),
+       another_approving_team:    create(:approving_team)
+    }
+
+
+    @users = {
+      manager_user:                         find_or_create(:disclosure_bmt_user),
+      approver_user:                        find_or_create(:disclosure_specialist),
+      another_approver_user:                create(:approver, approving_team: another_approving_team),
+      responder_user:                       create(:responder, responding_teams:[responding_team]),
+      another_responder_in_same_team_user:  create(:responder, responding_teams:[responding_team]),
+      another_responder_in_diff_team_user:  create(:responder),
+      press_officer_user:                   find_or_create(:press_officer),
+      private_officer_user:                 find_or_create(:private_officer)
+    }
+
+    @user_teams = {
+        manager:                          [ manager_user, disclosure_bmt_team ],
+        approver:                         [ approver_user, disclosure_team ],
+        another_approver:                 [ another_approver_user, another_approving_team ],
+        responder:                        [ responder_user, responding_team ],
+        another_responder_in_same_team:   [ another_responder_in_same_team_user, responding_team ],
+        another_responder_in_diff_team:   [ another_responder_in_diff_team_user, another_responder_in_diff_team_user.responding_teams.first ],
+        press_officer:                    [ press_officer_user, press_office_team ],
+        private_officer:                  [ private_officer_user, private_office_team ]
+    }
+
+    @cases = {
+       std_unassigned_foi:        create(:case),
+       std_awresp_foi:            create(:assigned_case,
```

Comment:
Took me a couple tries to realise that `aw` is `Awaiting` responder. Could we have a legend for what the shorthand means? I'd be happy to see proper shorthand here (anyone remember [Hungarian Notation](https://en.wikipedia.org/wiki/Hungarian_notation)??) so long as it's documented.

Reply:
have added explanation into standard setup where they are defined